Makefile

```diff
@@ -114,6 +114,9 @@ lint-go:
 
 lint: lint-go lint-js
 
+dump-test-schema:
+	docker-compose exec -T mysql_test mysqldump -uroot -ptoor schemadb --compact --skip-comments > server/datastore/mysql/schema.sql
```

Comment:
If we merge this, now this will be needed every time we change the schema. Not sure if this is in an ideal scenario yet, but I think it's better even if it takes one more step for devs.

Reply:
Let's make sure this is documented in https://github.com/fleetdm/fleet/blob/main/docs/3-Contributing/3-Migrations.md if we go ahead with this.

Reply:
Will do. I realized I need to tweak this slightly, will update this soon.

Reply:
I updated this to update the schema before running the tests, to make sure tests run against the latest schema. It's not ideal that a test modifies a file that other tests uses. I could make this a cmd instead and make the test not modify the schema, just test the migrations.